clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -105,6 +105,23 @@ public class ConsumerConfig extends AbstractConfig {
     public static final String HEARTBEAT_INTERVAL_MS_CONFIG = CommonClientConfigs.HEARTBEAT_INTERVAL_MS_CONFIG;
     private static final String HEARTBEAT_INTERVAL_MS_DOC = CommonClientConfigs.HEARTBEAT_INTERVAL_MS_DOC;
 
+    /**
+     * <code>group.protocol</code>
+     */
+    public static final String GROUP_PROTOCOL_CONFIG = "group.protocol";
+    public static final String DEFAULT_GROUP_PROTOCOL = GroupProtocol.GENERIC.name().toLowerCase();
+    public static final String GROUP_PROTOCOL_DOC = "The rebalance protocol consumer should use.  We currently " +
```

Comment:
Maybe we should stick with "group protocol" and not use the synonymous "rebalance protocol" here, although I know we use this interchangeably.

Also, maybe that's just the non-native speaker in me, but for me the sentence would be easier to parse with an article for consumer.

Reply:
+1